lib/Makefile

```diff
@@ -21,6 +21,8 @@ USB_SRCS := \
 
 SRCS += \
 	$(wildcard $(NXDK_DIR)/lib/xboxrt/*.c) \
+	$(shell find $(NXDK_DIR)/lib/xlibc/src/ -name "*.c") \
+	$(shell find $(NXDK_DIR)/lib/xlibc/src/ -name "*.s") \
```

Comment:
why is this done differently than all the others?

Reply:
All other libs have a single directory containing all source files, so wildcard is enough for them. For xlibc, I have several subdirectories (xlibc/src/assert, xlibc/src/setjmp) plus additional source files that don't reside in a subdirectory (not part of this PR), so need to look up these source files recursively. Afaik doing this with wildcard is possible but far uglier than using find.

For the future it might be better to list each file individually (like UBS_SRC) as looking files up on each run of make can be quite slow.

Reply:
Yes, I also agree regarding listing files manually. Not only is the wildcard potentially slower, but it might even include files we don't want to include (like files accidentally left behind, not managed by git).

I'm fine with this solution for now. It would be nice if you could create an issue about listing files manually (I assume other maintainers agree).

Reply:
We can do this later. Someone create an issue please.